packages/editor/src/components/editor-help/style.scss

```diff
@@ -1,3 +1,22 @@
+.button {
+	margin: 0.5rem 0.25rem;
+	padding: 0.75rem;
+	align-items: center;
+	justify-content: center;
+	border-radius: $border-width * 6;
```

Comment:
👋 @twstokes ! Noticed that elsewhere, the radius is defined as [$border-width * 4](https://github.com/WordPress/gutenberg/blob/ffcce3efd956672ffb1b31927b86ea4099782312/packages/block-library/src/file/style.native.scss#L6). I don't have a particular pref on which radius is better so, let's make sure @kyleaparker has the chance for a design review on the PR, thanks!

Reply:
Will do @hypest! I'd noticed this as well and settled on `6` because it matched her mockup closer, but agreed - I'll be sure to get her input.

Reply:
I think it's 8 in the mockups, but I might be off as I'm using some older Figma components. Should match the other button styles in the app :)

Reply:
Thank you both! I've adjusted this to match `$border-width * 4`, so I'll resolve this conversation. @kyleaparker I'll work to get things ready for a design review so you can have a complete look.